uni/lib/generated/intl/messages_pt_PT.dart

```diff
@@ -246,6 +246,7 @@ class MessageLookup extends MessageLookupByLibrary {
         "widget_prompt": MessageLookupByLibrary.simpleMessage(
             "Escolhe um widget para adicionares à tua área pessoal:"),
         "year": MessageLookupByLibrary.simpleMessage("Ano"),
-        "yes": MessageLookupByLibrary.simpleMessage("Sim")
+        "yes": MessageLookupByLibrary.simpleMessage("Sim"),
+        "tooltip":MessageLookupByLibrary.simpleMessage("Adicionar à Àrea Pessoal")
```

Comment:
Just two things:

- Make it a clear name, not only `tooltip`, add something more.
- It is `à Área`, not `à Àrea`